.gitignore

```diff
@@ -112,3 +112,5 @@ docs/index.rst
 
 # exclude project specific files
 pyuvsim/GIT_INFO
+
+\.idea/
```

Comment:
What is this?

Reply:
ugh. that's my editor's project files -- should I remove it and just manually ignore them? Do we want to keep the .gitignore clean?

Reply:
You should put that kind of thing in your global .gitignore (so they'll be ignored across all the repos on your machine)

Reply:
Oh yeah, I'd never thought of doing that before! Fixed in 4be03e7abad384b98118f5e6d47c37a9fcfd43be